src/main/java/io/fabric8/elasticsearch/plugin/OpenshiftRequestContextFactory.java

```diff
@@ -83,6 +83,10 @@ public OpenshiftRequestContext create(final RestRequest request, final UserProje
         Set<String> projects = new HashSet<>();
         boolean isClusterAdmin = false;
         String user = utils.getUser(request);
+        if(user.contains("\\")){
+            user = user.replaceAll("\\\\", "/");
```

Comment:
We really need to escape this much? Is there a java util we can use to escape for us so this looks cleaner?